assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -85,11 +86,14 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
 						'woo-gutenberg-products-block'
 					) }
 					checked={ isShippingCalculatorEnabled }
-					onChange={ () =>
+					onChange={ () => {
+						recordEditorEvent( 'cart_shipping_calculator_toggled', {
```

Comment:
Do we have a standard in mind for event names? Good to get that in place asap.

Reply:
Good point – I hadn't checked this against guidelines. There are clear guidelines in Field Guide. In d414e28e9c81c3dd18ce225758705adf12f176bc I've updated the event name per guidelines.

- `wcadmin_` service context is provided by core tracks
- `blocks_` context associates this event with Woo Blocks repo/plugin
- `cart_` - this is a cart block event
- `settings_` in the block settings area of the UI
- `shipping_calculator_` the shipping calculator setting
- `toggle` the user action (present tense as per guidelines)

So the full event name in this case is `wcadmin_blocks_cart_settings_shipping_calculator_toggle`.